psi4/src/psi4/libmints/eribase.cc

```diff
@@ -1921,6 +2019,19 @@ void TwoElectronInt::init_shell_pairs12() {
     pairs12_ = new ShellPair *[basis1()->nshell()];
     for (i = 0; i < basis1()->nshell(); ++i) pairs12_[i] = new ShellPair[basis2()->nshell()];
 
+    //  Store screened shell pairs in a vector of vectors
+    for (i = 0; i < basis1()->nshell(); ++i) {
+        std::vector<ShellPairScreen> row;
+        screenpairs12_.push_back(row);
+    }
+
+    // Total number of primitive pairs in all shell pairs (before screening) [FOR PRINING]
+    int total_prim_pairs = 0;
+    // Criteria for determining of overlap between primitives is significant for screening [FOR PRINTING]
+    double overlap_cutoff = 1.0e-12;
```

Comment:
This is probably something we would allow the user to change in their input.

Reply:
+1

Reply:
I think (in a later PR) we should set the default to 1e-10, which will require changing some test cases it seems

Reply:
I initially set the cutoff to 10e-10, but a few of the quick tests failed when checking SCF energy to 10 digits

Reply:
It looks like there's already something like this already,:
https://github.com/psi4/psi4/blob/637dbab34d0313a8cc81ea2d4c3740d07ff54328/psi4/src/read_options.cc#L1231
Although it's not clear to me how `INTS_TOLERANCE` should be used for screening